Magento2/Sniffs/Functions/DiscouragedFunctionSniff.php

```diff
@@ -20,13 +19,19 @@ class DiscouragedFunctionSniff extends ForbiddenFunctionsSniff
      */
     protected $patternMatch = true;
 
+    /**
+     * If true, an error will be thrown; otherwise a warning.
+     *
+     * @var boolean
+     */
+    public $error = false;
```

Comment:
Please validate the list against https://stackoverflow.com/questions/3115559/exploitable-php-functions and ask security team to review.

Reply:
Added.